src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -7,7 +7,16 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+
+    public static double sqrt(double number) throws IllegalArgumentException {
```

Comment:
java.lang.IllegalArgumentException is a child of RuntimeException. Should we add it to method signature?